package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "buildah-build",
-  "version": "1.0.0",
+  "version": "3.0",
```

Comment:
Probably we would not want to update this version. As there seems to be no breaking changes.

Reply:
In regards to that, I simply followed the recommendation/best practices that GitHub provided themselves for Actions migrating from Node 12 to Node 16.

> As a result, actions authors who move from targeting Node.js 12 to Node.js 16 should release with a new major version number to communicate that there is a possible breaking change for GitHub Enterprise Server 3.3 and earlier.

https://github.blog/changelog/2022-05-20-actions-can-now-run-in-a-node-js-16-runtime/

Reply:
If you disagree with that notion I will revert to v2 no problem

Reply:
@k3rnelpan1c-dev let us go with version `2.0.0` for now.

Reply:
Yes, please.